### PR TITLE
Fix Title parse

### PR DESCRIPTION
**Pull request by a632079**
image
Like that, it is not equel to `true`. so, i add `Array.isArray()`

### CODE REVIEW

File: library.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ plugin.getUsers = function (callback) {
 
 plugin.updateTitle = function (data, callback) {
 	translator.translate('[[2factor:title]]', function (title) {
-		if (data.templateData.url.match(/user\/.+\/2factor/)) {
+		if (Array.isArray(data.templateData.url.match(/user\/.+\/2factor/))) {
```
Comment (julianlam) [inline library.js]: Use of `Array.isArray` is potentially confusing, can you please remove that and instead use `!!(data.templateData.url.match(/user\/.+\/2factor/).length)`?

Comment (julianlam) [reply library.js]: I think even the `!!` is not needed, and can be excluded.

**Comment by a632079** [comment]
I'm SORRY that I'm wrong. I have searched it in https://developer.mozilla.org/en-US/docs/Glossary/Truthy.
`[] //Array` is equel to `truthy` .
BUT I'm confused why `Array` is not equel to `True //Boolen`. Could you give me some tips?

**Comment by julianlam** [comment]
I believe a regular expression match is not necessarily an array (though it is array-like)

**Pull request closed without merging**